brainstormforce/astra
# AST-2233 - Default style seperator block displays "Wide Line" even when default with templates

### Description

On some templates:
BugFix: The separator shows full wide in the editor.
BugFix: The separator does not align in center in frontend.
Jira: https://brainstormforce.atlassian.net/browse/AST-2233

### Screenshots

https://d.pr/v/vzS5uC

### Types of changes


BugFix: The separator shows full wide in the editor.
BugFix: The separator does not align in center in frontend.



### How has this been tested?


- On fresh installation, install Astra, install Astra Addon, install Spectra.
- Pick any template (Life Coach, Construction Company etc).
- Create a page using Gutenberg.
- Add a seperator block (Default will be selected by default).
- The width should not be “Wide Line“ or “Dotted“ but the line should have lesser width (as shown in [Annotation on 2022-09-23 at 18-00-26.png](https://d.pr/i/AdxjX1)).

Test in frontend too, should exactly be same as editor.
### Checklist:
